src/ArticleType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace RA;
+
+abstract class ArticleType
+{
+    public const Game = 1;
+
+    public const Achievement = 2;
+
+    public const User = 3;
+
```

Comment:
I didn't find any usage of the value 4 but it may exist.

Reply:
It probably used to exist... I think maybe it was "News"? I'm not sure, sorry

Reply:
Good catch! After a 2nd look, I found that the enum already exists under `SubjectType` where the value 4 was defined.
I've just reworked my PR in order to merge the 2 enums.
Thank for the review :D